### ISSUE TITLE
Self-hosted API

### ISSUE THREAD

**Issue by 7flash** [getcursor/cursor#360]

**Comment by truell20** [comment]
Definitely interested in making this available if folks want! would you self host an instance for yourself?

**Comment by 7flash** [comment]
Of course, locally! Ideally, do you think, it's possible to make server process built into app executable? Otherwise, running a Docker container works too.

**Comment by danperks** [comment]
Closing due to inactivity
